cl/alerts/views.py

```diff
@@ -177,32 +177,96 @@ def subscribe_docket_alert(
     request: HttpRequest, secret_key: str
```

Comment:
Could you just switch to a single urls.py path that has `/(un)?subscribe/` in it, and then eliminate combine `subscribe_docket_alert` with `unsubscribe_docket_alert` and `toggle_docket_alert_confirmation`? It feels like you're doing routing in your view.

Reply:
Thanks, this worked well so now we just have one path: `toggle_docket_alert_confirmation`